client/me/reauth-required/two-factor-actions.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import React, { useCallback } from 'react';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEventWithClientId } from 'calypso/state/analytics/actions';
+import './two-factor-actions.scss';
+
+interface TwoFactorActionsProps {
+	isAuthenticatorSupported: boolean;
+	isSecurityKeySupported: boolean;
+	isSmsSupported: boolean;
+	isSmsAllowed: boolean;
+	onChange: ( authType: string ) => void;
+	twoFactorAuthType: string;
+}
+
+export function TwoFactorActions( {
+	isAuthenticatorSupported,
+	isSecurityKeySupported,
+	isSmsSupported,
+	isSmsAllowed,
+	onChange,
+	twoFactorAuthType,
+}: TwoFactorActionsProps ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const recordButtonClicked = useCallback(
```

Comment:
The name of this handler was already a bit confusing: the name suggests that it only records the tracks event, but it does the entire handling, including calling `onChange`. Better one would be `handleButtonClick`.

Reply:
> The name of this handler was already a bit confusing: the name suggests that it only records the tracks event, but it does the entire handling, including calling `onChange`. Better one would be `handleButtonClick`.

Renamed to `handleButtonClick` in c78cb510bc